cura/UI/WelcomePagesModel.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2019 Ultimaker B.V.
```

Comment:
All renamed, I left most model files in `cura.UI` without further reorganization because I think many of them should be reviewed for possible refactoring.